package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "accessible-menu",
-  "version": "3.0.0",
+  "version": "3.0.1-alpha",
```

Comment:
Don't change the version. The release script will take care of this.

Reply:
Sorry for that! I will restore it in the next commit.